vault.js

```diff
@@ -1,4 +1,17 @@
 'use strict';
 module.exports = function() {
-  
+    var fart={};
+    function setValue(key, value){
+        fart[key] = value;
+    }
+    function getValue(key){
+        if (fart.hasOwnProperty(key)){
+            return fart[key];
+        }else{
+            return null;
+        }
+    
+    }
+       return {getValue,setValue}
```

Comment:
The `return` is a peer of the var and function and should not be indented extra.